shared/actions/chat/inbox.js

```diff
@@ -986,7 +986,10 @@ function _joinConversation(action: ChatGen.JoinConversationPayload) {
 
 function _previewChannel(action: ChatGen.PreviewChannelPayload) {
   const convID = Constants.keyToConversationID(action.payload.conversationIDKey)
-  return Saga.call(RPCChatTypes.localPreviewConversationByIDLocalRpcPromise, {convID})
+  return Saga.sequentially([
+    Saga.call(RPCChatTypes.localPreviewConversationByIDLocalRpcPromise, {convID}),
+    Saga.put(ChatGen.createSelectConversation({conversationIDKey: action.payload.conversationIDKey, fromUser: true}))
```

Comment:
:sheep: do you think this should go in `manage-channels/container.js @_onPreview` so this saga can just call the rpc?

Reply:
no, because this needs to happen sequentially. If you select the conversation before that rpc finishes it won't work.